examples/sklearn/iris-classifier/README.md

```diff
@@ -0,0 +1,308 @@
+# Deploy a scikit-learn model as a web service
+
+This example shows how to deploy a classifier trained on the famous [iris data set](https://archive.ics.uci.edu/ml/datasets/iris) using scikit-learn.
+
+<br>
+
+## Train your model
+
+1. Create a Python file `trainer.py`.
+2. Use scikit-learn's `LogisticRegression` to train your model.
+3. Add code to pickle your model (you can use other serialization libraries such as joblib).
+4. Upload it to S3 (boto3 will need access to valid AWS credentials).
+
+```python
+import boto3
+import pickle
+from sklearn.datasets import load_iris
+from sklearn.model_selection import train_test_split
+from sklearn.linear_model import LogisticRegression
+
+# Train the model
+
+iris = load_iris()
+data, labels = iris.data, iris.target
+training_data, test_data, training_labels, test_labels = train_test_split(data, labels)
+
+model = LogisticRegression(solver="lbfgs", multi_class="multinomial")
+model.fit(training_data, training_labels)
+accuracy = model.score(test_data, test_labels)
+print("accuracy: {:.2f}".format(accuracy))
+
+# Upload the model
+
+pickle.dump(model, open("model.pkl", "wb"))
+s3 = boto3.client("s3")
+s3.upload_file("model.pkl", "my-bucket", "sklearn/iris-classifier/model.pkl")
+```
+
+Run the script locally:
+
+```bash
+# Install scikit-learn and boto3
+$ pip3 install sklearn boto3
+
+# Run the script
+$ python3 trainer.py
+```
+
+<br>
+
+## Define a predictor
+
+1. Create another Python file `predictor.py`.
+2. Add code to load and initialize your pickled model.
+3. Add a prediction function that will accept a sample and return a prediction from your model.
+
+```python
+# predictor.py
+
+import pickle
+import numpy
+
+
+model = None
+labels = ["iris-setosa", "iris-versicolor", "iris-virginica"]
+
+
+def init(model_path, metadata):
+    global model
+    model = pickle.load(open(model_path, "rb"))
+
+
+def predict(sample, metadata):
+    input_array = numpy.array(
+        [
+            sample["sepal_length"],
+            sample["sepal_width"],
+            sample["petal_length"],
+            sample["petal_width"],
+        ]
+    )
+
+    label_id = model.predict([input_array])[0]
+    return labels[label_id]
+```
+
+<br>
+
+## Specify Python dependencies
+
+Create a `requirements.txt` file to specify the dependencies needed by `predictor.py`. Cortex will automatically install them into your runtime once you deploy:
+
+```python
+# requirements.txt
+
+numpy
+```
+
+You can skip dependencies that are [pre-installed](../../../docs/deployments/predictor.md#pre-installed-packages) to speed up the deployment process. Note that `pickle` is part of the Python standard library so it doesn't need to be included.
```

Comment:
Sorry, I got confused when I commented on this yesterday; since this will be read in GitHub, I think it should point to `https://www.cortex.dev/deployments/predictor#pre-installed-packages`